planetarypraxis/smartforests
# fix: prevent tag duplication on pages and add deduplicate_tags command

## Description
When pages were saved, they were given duplicate tags (e.g. "datafication" (en) and "datafication" (fr)), because of how the TagField component works. This PR updates the custom TagFieldWidget to remove these duplicates.

This PR also adds a command, deduplicate_tags, to remove this duplication from all pages.

## How Can It Be Tested?
On the main branch, create a translated version of a tag which is identical to the original. Then, add this tag to a Story. There should now be duplication visible on the Story page sidebar.

Switch to the PR branch and re-publish the Story. The duplication should be gone.

Switch back to main and re-publish the Story. The duplication should be back.

Finally, switch back to the PR branch and run `python manage.py deduplicate_tags` and check the duplication is gone again.

## How Will This Be Deployed?
Standard deployment, then we run the deduplicate_tags command on live.

## Types of changes

## Checklist:


- I've checked the spec (e.g. Figma file) and documented any divergences.
- My change requires a change to the documentation.
- I've updated the documentation accordingly.
- Replace unused checkboxes with bullet points.
